### PR TITLE
Docs for 4.4

### PR DESCRIPTION
**Pull request by benarent**
- [x] Forked Docs
- [x] Fixed switcher
- [x] bumped version refs
- [x] Ported over Andrews notes about RBAC for node labels. 

Once merged, I'll add in session streaming and nots on concurrent session control

### CODE REVIEW

**Comment by benarent** [comment]
retest this please

File: docs/4.4/config-reference.md [hunk 1]
```diff
@@ -0,0 +1,329 @@
...
+
+        # Use this setting to configure teleport to store the recorded sessions in
+        # an AWS S3 bucket or use GCP Storage with 'gs://'. See "Using Amazon S3"
+        # chapter for more information.
+        audit_sessions_uri: 's3://example.com/path/to/bucket?region=us-east-1'
+
+    # CA Signing algorithm used for OpenSSH Certificates
+    # Defaults to rsa-sha2-512 in 4.3 and above.
+    # valid values are: ssh-rsa, rsa-sha2-256, rsa-sha2-512; ssh-rsa is SHA1
+    ca_signature_algo: “rsa-sha2-512”
```
Comment (webvictim) [inline docs/4.4/config-reference.md:88]:
```suggestion
    ca_signature_algo: "rsa-sha2-512"
```
(remove 'smart quotes')

File: docs/4.4/config-reference.md [hunk 2]
```diff
@@ -0,0 +1,329 @@
...
+        # second_factor can be off, otp, or u2f
+        second_factor: otp
+        # this section is used if second_factor is set to 'u2f'
+        u2f:
+            # app_id must point to the URL of the Teleport Web UI (proxy) accessible
+            # by the end users
+            app_id: https://localhost:3080
+            # facets must list all proxy servers if there are more than one deployed
+            facets:
+
```
Comment (webvictim) [inline docs/4.4/config-reference.md:157]:
```suggestion
```

File: docs/4.4/enterprise/ssh-kubernetes-fedramp.md [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+  enabled: false
+```
+
+### Teleport Node
+
+Save the following configuration file as `/etc/teleport.yaml` on the node server.
+```yaml
+teleport:
+  auth_token: zw6C82kq7VEUSJeSDzuldWsxakql6jrTYmphxRQOlrATTGbLQoaIwEBo48o9
+
```
Comment (webvictim) [inline docs/4.4/enterprise/ssh-kubernetes-fedramp.md:97]:
```suggestion
```

File: docs/4.4/enterprise/ssh-kubernetes-fedramp.md [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+  auth_servers: [ "10.1.1.10:3025" ]
+
+  # enable ssh service and disable auth and proxy:
+ssh_service:
+  enabled: true
+
```
Comment (webvictim) [inline docs/4.4/enterprise/ssh-kubernetes-fedramp.md:103]:
```suggestion
```

File: docs/4.4/enterprise/sso/ssh-adfs.md [hunk 5]
```diff
@@ -0,0 +1,204 @@
...
+based access control (RBAC) it allows SSH administrators to define policies
+like:
+
+* Only members of "DBA" group can SSH into machines running PostgreSQL.
+* Developers must never SSH into production servers.
+* ... and many others.
+
+!!! warning "Version Warning"
+
+
```
Comment (webvictim) [inline docs/4.4/enterprise/sso/ssh-adfs.md:23]:
```suggestion
```

**Review by webvictim — COMMENTED** [review]
I had a very quick look and fixed a few issues.

File: docs/4.4.yaml [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+        #    - OneLogin:  enterprise/sso/ssh-one-login.md
+        #    - OIDC:  enterprise/sso/oidc.md
+        #    - Okta:  enterprise/sso/ssh-okta.md
+        - Approval Workflow: enterprise/workflow/index.md
+        #- Teleport Plugins:
+        #    - Teleport  Plugin Setup: enterprise/workflow/ssh-approval-jira-cloud.md
+        #    - Teleport Jira Server Plugin Setup: enterprise/workflow/ssh-approval-jira-server.md
+        #    - Teleport Mattermost Plugin Setup: enterprise/workflow/ssh-approval-mattermost.md
+        #    - Teleport Pagerduty Plugin Setup: enterprise/workflow/ssh-approval-pagerduty.md
+        #    - Teleport Slack Plugin Setup: enterprise/workflow/ssh-approval-slack.md
```
[the remark this answers is not in the document]
Comment (benarent) [reply docs/4.4.yaml:77]: We are going to moving these to a new IA for Teleport 5 docs. For now, it's a handy comment reference for me when I  build it out. We won't be doing this in the next version of the docs.

File: docs/4.4.yaml [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+        - Introduction: enterprise/introduction.md
+        - Quick Start Guide: enterprise/quickstart-enterprise.md
+        - Single sign-on (SSO): enterprise/sso/ssh-sso.md
+         #- SSO Guides:
+        #    - Azure Active Directory (AD): enterprise/sso/ssh-azuread.md
+        #    - Active Directory (ADFS):  enterprise/sso/ssh-adfs.md
+        #    - G Suite:  enterprise/sso/ssh-gsuite.md
+        #    - OneLogin:  enterprise/sso/ssh-one-login.md
+        #    - OIDC:  enterprise/sso/oidc.md
+        #    - Okta:  enterprise/sso/ssh-okta.md
```
[the remark this answers is not in the document]
Comment (benarent) [reply docs/4.4.yaml:70]: Commented on the other note.

File: docs/4.4.yaml (hunk 6 above)
[the remark this answers is not in the document]
Comment (webvictim) [reply docs/4.4.yaml:77]: OK.

**Review by webvictim — APPROVED** [review]

**Review by webvictim — COMMENTED** [review]
Too big to review really so I'm just going with it!

**Pull request merged**